core/src/main/java/org/apache/accumulo/core/file/blockfile/cache/BlockCache.java

```diff
@@ -17,10 +17,30 @@
  */
 package org.apache.accumulo.core.file.blockfile.cache;
 
+import org.apache.accumulo.core.conf.AccumuloConfiguration;
+
 /**
  * Block cache interface.
  */
 public interface BlockCache {
+
+  /**
+   * Start the block cache
+   *
+   * @param conf
+   *          Accumulo configuration object
+   * @param maxSize
+   *          maximum size of the on-heap cache
+   * @param blockSize
+   *          size of the default RFile blocks
+   */
+  void start(AccumuloConfiguration conf, long maxSize, long blockSize) throws Exception;
```

Comment:
Hmm, yeah, I later realized most impls would find some way to piggy-back properties into accumulo-site. Mostly, I figured that, eventually, we'd also have other "universal" properties (max size and block size) that all of the implementations would use.

Having some object there (or, using the Factory as Keith's suggestion pushes towards) would at least avoid runtime issues (where the new Accumulo expects a `start(AccumuloConfiguration, long, long, ...)` instead of just `start(AccumuloConfiguration, long, long)`.